src/bridge/lib/txn.js

```diff
@@ -155,6 +155,10 @@ const sendSignedTransaction = (web3, stx) => {
       .on('receipt', txn => {
         resolve(Just(Ok(txn.transactionHash)))
       })
+      .on('confirmation', (confirmationNum, txn) => {
+        confirmationCb(txn.transactionHash, confirmationNum + 1)
```

Comment:
Feels weird to have a callback like this inside a promise, but I think this allows us to implement the feature with minimal code changes to sendSignedTransaction.